ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -1,21 +1,21 @@
-{% if router_isis is defined and router_isis is not none %}
+{% if router_isis is arista.avd.defined %}
 
 ## Router ISIS
 
 ### Router ISIS Summary
 
 | Settings | Value |
 | -------- | ----- |
-{%     if router_isis.instance is defined %}
+{%     if router_isis.instance is arista.avd.defined %}
 | Instance | {{ router_isis.instance }} |
 {%     endif %}
-{%     if router_isis.net is defined %}
+{%     if router_isis.net is arista.avd.defined %}
 | Net-ID | {{ router_isis.net }} |
 {%     endif %}
-{%     if router_isis.is_type is defined %}
+{%     if router_isis.is_type is arista.avd.defined %}
 | Type | {{ router_isis.is_type }} |
 {%     endif %}
-{%     if router_isis.address_family is defined %}
+{%     if router_isis.address_family is arista.avd.defined %}
 | Address Family | {% for af in router_isis.address_family %}{{ af }} {% endfor %}|
```

Comment:
You can use join(', ') here instead of for loop. https://avd.sh/en/releases-v3.x.x/docs/contribution/style-guide.html#syntax-4-expand-list-on-single-line

Reply:
Fixed.